spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -375,6 +388,44 @@ def delete_app_group!(app_group_id)
       parse_response(r)
     end
 
+    #####################################################
+    # @!group Cloud Containers
+    #####################################################
+
+    def cloud_containers
+      paging do |page_number|
+        r = request(:post, 'account/cloudContainer/listCloudContainers.action', {
+            teamId: team_id,
+            pageNumber: page_number,
+            pageSize: page_size,
+            sort: 'name=asc'
+        })
+        result = parse_response(r, 'cloudContainerList')
+
+        csrf_cache[Spaceship::Portal::CloudContainer] = self.csrf_tokens
+
+        result
+      end
+    end
+
+    def create_cloud_container!(name, identifier)
+      ensure_csrf(Spaceship::Portal::CloudContainer)
+
+      r = request(:post, 'account/cloudContainer/addCloudContainer.action', {
+          name: valid_name_for(name),
+          identifier: identifier,
+          teamId: team_id
+      })
+      a = parse_response(r)
+
+      # If request fails, try to fetch container with identifier if it already exists
```

Comment:
Absolutely! I'm more than happy to contribute in the best way and not break any existing architectural logic 😊 
Before I do anything and since I'm new to this repo, would you say the error handling is best suited in Portal::CloudContainer.create or in Produce::CloudContainer.create?

Reply:
True!
I'll remove this and run through the flow again to see that everything works as expected.

Reply:
Ok, removed the fallback and run through the tests and everything works as expected from my point of view! 😊